lumicks/pylake/file.py

```diff
@@ -26,8 +26,10 @@ class File(Group, Force, DownsampledFD, BaselineCorrectedForce, PhotonCounts, Ph
     filename : str | os.PathLike
         The HDF5 file to open in read-only mode
 
-    rgb_to_detectors : Dict[Color, str]
-        Dictionary that maps RGB colors to a photon detector channel (either photon counts, or photon time tags)
+    rgb_to_detectors : Optional[Dict[Color, str]]
```

Comment:
The color ended up being just a `str`
```suggestion
    rgb_to_detectors : Optional[Dict[str, str]]
```

Reply:
👍 

I added one extra check to verify that the keys are correct. Otherwise you could accidentally use an incorrect capitalization like "red" and wonder why it doesn't work.